samples/nrf9160/modem_shell/src/th/th_ctrl.c

```diff
@@ -264,11 +291,24 @@ static void th_ctrl_data_start(struct th_ctrl_data *data,
 
 void th_ctrl_start(const struct shell *shell, size_t argc, char **argv, bool is_background)
 {
-	/* Only iperf3 currently supported */
-	if (strcmp(argv[1], "iperf3") != 0) {
-		mosh_error("Only iperf3 is supported currently.");
+	bool start = false;
+
+	/* Only iperf3 and ping are currently supported */
+#if defined(CONFIG_MOSH_IPERF3)
+	if (!strcmp(argv[1], "iperf3")) {
+		start = true;
+	}
+#endif
+#if defined(CONFIG_MOSH_PING)
+	if (!strcmp(argv[1], "ping")) {
+		start = true;
+	}
+#endif
+	if (!start) {
+		mosh_error("Cannot start a thread: command %s", argv[1]);
 		return;
 	}
+	/* TODO: there cannot be iperf3 and ping running at same time at the same context */
```

Comment:
Is this comment here intentionally or should this check be added to this PR?

Reply:
Implementation time decision was that no check for that is done, limitations are noted in related shell cmd help. Let that be enough. I'll remove this one.

Reply:
Comment was removed.